src/bindings/html/dom.js

```diff
@@ -42,15 +42,14 @@ export function getAttributes(element) {
 }
 
 function getTranslatables(element) {
-  const nodes = [];
+  const nodes = [...element.querySelectorAll('*[data-l10n-id]')];
 
-  if (typeof element.hasAttribute === 'function' &&
+  if (element.nodeType !== element.DOCUMENT_FRAGMENT_NODE &&
```

Comment:
Why this change?  The original served the same purpose and was more generic and possibly future-proof.